lib/slack/real_time/concurrency/celluloid.rb

```diff
@@ -38,6 +38,8 @@ def run_loop
           rescue EOFError => e
             logger.debug("#{self.class}##{__method__}") { e }
             driver.emit(:close, WebSocket::Driver::CloseEvent.new(1001, 'server closed connection')) unless @closing
+          ensure
+            terminate
```

Comment:
@dblock I have submitted the PR: https://github.com/slack-ruby/slack-ruby-client/pull/106